config/samples/resources/cloudidsendpoint/cloudids_v1beta1_cloudnetwork.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeNetwork
+metadata:
+  name: cloudidsendpoint-dep1
```

Comment:
Actually all the dependent resources of different kinds share the same name `cloudidsendpoint-dep`.

The reason is that the resource name can be associated to the sample and it can be generated automatically without knowing other dependencies.